tests/h/traversal/contexts_test.py

```diff
@@ -385,42 +233,9 @@ def test_default_property_if_default_organization(self, factories, pyramid_reque
         assert organization_context.default is True
 
 
-@pytest.mark.usefixtures('user_service')
-class TestUserRoot(object):
-
-    def test_it_fetches_the_requested_user(self, pyramid_request, user_factory, user_service):
-        user_factory["bob"]
-
-        user_service.fetch.assert_called_once_with("bob", pyramid_request.authority)
-
-    def test_it_raises_KeyError_if_the_user_does_not_exist(self,
-                                                           user_factory,
-                                                           user_service):
-        user_service.fetch.return_value = None
-
-        with pytest.raises(KeyError):
-            user_factory["does_not_exist"]
-
-    def test_it_returns_users(self, factories, user_factory, user_service):
-        user_service.fetch.return_value = user = factories.User.build()
-
-        assert user_factory[user.username] == user
-
-    @pytest.fixture
-    def user_factory(self, pyramid_request):
-        return UserRoot(pyramid_request)
-
-    @pytest.fixture
-    def user_service(self, pyramid_config):
-        user_service = mock.create_autospec(UserService, spec_set=True, instance=True)
-        pyramid_config.register_service(user_service, name='user')
-        return user_service
-
-
-@pytest.fixture
-def organizations(factories):
-    # Add a handful of organizations to the DB to make the test realistic.
-    return [factories.Organization() for _ in range(3)]
+class FakeGroup(object):
```

Comment:
One day I'd love to see all uses of `Fake*` in our tests go the way of the dodo (I know this one is legacy, not new!).

Reply:
Me too